src/Avalonia.Base/Rendering/Composition/Compositor.cs

```diff
@@ -136,5 +144,28 @@ internal void InvokeBeforeNextCommit(Action<Task> action)
             _invokeBeforeCommit.Enqueue(action);
             RequestCommitAsync();
         }
+
+        /// <summary>
+        /// Attempts to query for a feature from the platform render interface
+        /// </summary>
+        public ValueTask<object?> TryGetRenderInterfaceFeature(Type featureType)
```

Comment:
Can there be a situation, when compositor can't be started or failed, but developer wants to read call TryGetRenderInterfaceFeature? _pendingServerCompositorJobs won't be processed, task will never end.
If it is possible, should this method throw an exception early?

Reply:
Compositor always subscribes to a render loop in its constructor. Once we implement unsubscription there will be a more generic API to enqueue render thread tasks which would subscribe to the render loop again.